tethys/DataReader/HistPopIrrData.py

```diff
@@ -64,6 +67,51 @@ def getIrrYearData(settings):
     return irr
 
 
+def getIrrYearData_Crops(settings):
```

Comment:
We are now trying to maintain a consistent formatting style to our Python development.  Functions, methods, and variables should be snake case (e.g. get_irr_year_data_crops(settings)...).  Functions and methods should have a docstring formatted like:
```python
def get_irr_year_data_crops(settings):
    """This part contains a statement about what the function does.
    
    :param settings:                 This is the expected type of what is being passed; This is a description of the object that is being passed and states the default value if there is one.
    :returns:                        This is a description of what is being returned if applicable
    """
```